### PR TITLE
Update ui-testing-considerations.md

### PR DESCRIPTION
**Pull request by Ahetejaz**
The existing content is confusing and gives an impression that Visible UI Testing is supported by Microsoft-hosted agent. Microsoft-hosted agents do not support Visible UI Test, neither it can run visible UI test for desktop application.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@wisdeom 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Ahetejaz** [comment]
@wisdeom Hi, did you get a time to review these changes. Thank you.

**Comment by Jak-MS** [comment]
@juliakm - perhaps you can take a look?  or we can wait for @wisdeom.

**Comment by Ahetejaz** [comment]
@Jak-MS @wisdeom @Battle-born @juliakm Hi Team, it seems we lost  a track on this one. Did you guys get bandwidth to review this change. Current description creates a confusion around ability of UI Visible execution on MS Hosted agents.

Thank you.

**Comment by Jak-MS** [comment]
@Battle-born - can you take a look and approve if these changes are okay to publish?

File: docs/pipelines/test/ui-testing-considerations.md [hunk 1]
```diff
@@ -66,8 +66,10 @@ A special configuration is required for agents to run UI tests in visible UI mod
 
 ### Visible UI testing using Microsoft-hosted agents
 
-Microsoft-hosted agents are preconfigured for UI testing and UI tests for both
-web apps and desktop apps. Microsoft-hosted agents are also
+Visible UI Testing in not supported in Microsoft-hosted agents. The test would fail with error message "element not interactable". Microsoft-hosted agents support Headless mode UI Testing.
```
Comment (juliakm) [inline docs/pipelines/test/ui-testing-considerations.md:69]:
```suggestion
Visible UI testing in not supported in Microsoft-hosted agents. The test fails with error message "element not interactable". Microsoft-hosted agents support headless browser testing.
```

**Comment by juliakm** [comment]
I made a couple of small changes. This looks good to me. 

#sign-off

**Pull request merged**